resources/event-hackathon/notebook-authoring-checklist.md

```diff
@@ -0,0 +1,120 @@
+# Notebook authoring accessibility checklist
+
+How usable Jupyter notebooks are for disabled people can vary greatly depending on how they are set up and presented. Writing notebook content more accessibly is a great place for notebook authors to start making changes now!
+
+Use this checklist to review notebooks and as a prompt to ask more questions about how a notebook's content can be accessed.
+
+This checklist is based on the [draft authoring tips from the Notebooks For All team](accessibility-tips-for-jupyter-notebooks.md).
+
+**By the Notebooks for All team** - Erik Tollerud, Isabela Presedo-Floyd, Jenn Kotler, Patrick Smyth, Tony Fast
```

Comment:
Let's re-order by who worked on this document the most:
Isabela Presedo-Floyd, Jenn Kotler, Patrick Smyth, Tony Fast, Erik Tollerud